index.html

```diff
@@ -186,73 +186,72 @@ <h3 class="mt-post">Multi-Post-Stories</h3>
             </div>
           </div>
         </div>
-      </section>
-      <!-- about me -->
-      <section class="about-me bg-white">
-        <div class="first-section">
-          <h3>About Myself</h3>
-          <p>
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
-          </p>
-          <div>
-            <p><a href="#">LET'S CONNECT</a></p>
-          </div>
-        </div>
-        <div class="icons">
-          <ul>
-            <li>
-              <a href="#"><img src="images/Vector-1.svg" alt="twitter" /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img src="images/Linkedinicon.svg" alt="LinkedIn"
-              /></a>
-            </li>
-            <li>
-              <a href="#"><img src="images/Vector-2.svg" alt="Mark" /></a>
-            </li>
-            <li>
-              <a href="#"><img src="images/Vector-3.svg" alt="Github" /></a>
-            </li>
-            <li>
-              <a href="#"><img src="images/Vector.svg" alt="other" /></a>
-            </li>
-          </ul>
-          <div class="get-resume">
-            <a href="#">Get my resume</a>
-          </div>
-        </div>
-        <section class="second-section bg-white">
-          <div class="lang-dropdown">
-            <h3>Languages</h3>
-            <span class="dropdown"></span>
-          </div>
-          <div class="technologies">
-            <div class="tech">
-              <img src="images/icjscript.png" alt="dropdown" />
-              <p>Javascript</p>
+        <div id="section-5" class="contain5">
+          <div class="s5">
+            <div class="about-text">
+              <div class="head-text">
+                <h1>About Myself</h1>
+              </div>
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+              </p>
+            </div>
+            <div class="lc">LET'S CONNECT</div>
```

Comment:
- [ ] As the previous reviewer mentioned, `LET'S CONNECT` is a subtitle in this case, so it would be better if you use `hx` HTML tag instead, kindly correct that.  😁